src/script/E2EIdentity/EnrollmentTimer/EnrollmentTimer.ts

```diff
@@ -99,6 +100,11 @@ export function getEnrollmentTimer(
   const deadline = getGracePeriod(identity, e2eiActivatedAt, teamGracePeriodDuration);
   const nextTick = getNextTick(deadline);
 
+  // For the first activation, we want to trigger the timer immediately
+  if (isFirstActivation) {
+    return {isSnoozable: nextTick > 0, firingDate: Date.now()};
```

Comment:
question: Will it be snoozable for the existing mls client (basic credential type) during the grace period? 🤔

Reply:
Depends on the nextTick value I guess.

Nothing changed code related to this case. We just show the modal now with the option to snooze after first activation instead of snoozing automatically.